### DISCUSSION TITLE
How to Not Auto Focus First Element When Dialog is Opened?

### DISCUSSION THREAD

**Discussion by elson-currentcorp** [chakra-ui/ark#2624] [Q&A] [answered]
Whenever the dialog opens, it auto focuses the first input element inside the dialog. How do I prevent this behavior? 

The Dialog has a restoreFocus prop, but does not have an autoFocus prop?

I also tried setting initialFocusEl to `() => null` but it still focuses the first element.

**Comment by segunadebayo** [comment, marked as answer]
You can achieve that by setting the initial focus el to the dialog content, or set tabIndex=0 on it
